test/e2e/helpers.js

```diff
@@ -203,17 +203,26 @@ async function withFixtures(options, testSuite) {
       }
       const contracts =
         smartContract instanceof Array ? smartContract : [smartContract];
-      await Promise.all(
-        contracts.map((contract) => seeder.deploySmartContract(contract)),
-      );
+
+      const hardfork =
+        localNodeOptsNormalized[0].options.hardfork || 'muirGlacier';
+      for (const contract of contracts) {
+        await seeder.deploySmartContract(contract, hardfork);
+      }
```

Comment:
This loop has slightly different behavior than then previous `await Promise.all` way. This sends one transaction, waits for the transaction hash, then waits for the receipt for that transaction, _and then_ starts over for the next contract. The previous method send all of the transactions without waiting for anything, then waits all at once for all receipts.

Reply:
ah, interesting. I wonder if it is viem attempting to sign them but not accounting for pending nonces.